index.html

```diff
@@ -48,5 +48,193 @@ <h1 class="title">I’m Mariane<span id="info" class="title">Glad to see you!</s
+                            <p class="subTitle">Canopy</p>
+                            <ul class="infoContainer">
+                                <li class="role">Back End Dev</li>
+                                <li class="year">2015</li>
+                            </ul>
+                        </div>
+                        <p class="projectDescription">A daily selection of privately personalized reads;
+                             no accounts or sign-ups required.                  
+                        </p>
+                        <div class="tags">
+                            <ul class="tagsFlex">
+                                <li class="html">HTML</li>
+                                <li class="css">CSS</li>
+                                <li class="javascript">Javascript</li>
+                            </ul>
+                        </div>
+                        <button type="button" class="viewProject">
+                            See Project
+                        </button>
+                    </div>
+                </li>
+            </ul>
+
+            <ul>
+                <li class="project">
+                    <img src="src/Snapshoot-Portfolio3.png" alt="Snapshot3">
+                    <div class="description">
+                        <h2 class="workstitle">Multi-Post Stories</h2>
+                        <div class="info">
+                            <p class="subTitle">Canopy</p>
+                            <ul class="infoContainer">
+                                <li class="role">Back End Dev</li>
+                                <li class="year">2015</li>
+                            </ul>
+                        </div>
+                        <p class="projectDescription">A daily selection of privately personalized reads;
+                             no accounts or sign-ups required.                  
+                        </p>
+                        <div class="tags">
+                            <ul class="tagsFlex">
+                                <li class="html">HTML</li>
+                                <li class="css">CSS</li>
+                                <li class="javascript">Javascript</li>
+                            </ul>
+                        </div>
+                        <button type="button" class="viewProject">
+                            See Project
+                        </button>
+                    </div>
+                </li>
+            </ul>
+    
+        </div>
+    </section>
+
+    <section class="about-section" >
+            <h1 class="title">About<span>Myself</span>
+            </h1>
+    
+            <h6 class="paragraph">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.</h6>
+    
+            <h6 class="aboutline">LET’S CONNECT</h6>
```

Comment:
- [ ] Please make sure to follow [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) for about section too. Here you have a screenshot of how I am seeing it:
![image](https://user-images.githubusercontent.com/74506933/153707909-09ebc453-dfb8-459c-a4fb-8621820ed2b6.png)